### PR TITLE
infra: bump rust version

### LINKED ISSUE #2381: update rust version
probably to 1.76 coming out shortly

### PR DESCRIPTION
**Pull request by ad-tra**
fixes #2381

### CODE REVIEW

**Comment by Parth** [comment]
Unless it's intentional, I don't think you actually bumped the version file, there's a tool chain file in the root of the project. That's where you need to make the change if CI is happy and the version has been bumped then you're set.

File: rust-toolchain.toml
Comment (ad-tra) [inline rust-toolchain.toml:1]: @Parth do you mean this file?

Comment (Parth) [reply rust-toolchain.toml:1]: oh, idk why I didn't see this

Comment (Parth) [reply rust-toolchain.toml:1]: yeah this is good

**Pull request merged**